gateway/ln-gateway/src/lib.rs

```diff
@@ -1392,6 +1416,13 @@ impl Gateway {
     }
 }
 
+#[derive(serde::Serialize, serde::Deserialize)]
+pub struct GatewayBalances {
+    pub onchain_balance_sats: u64,
+    pub lightning_balance_msats: u64,
+    pub ecash_balance_msats: u64,
+}
```

Comment:
I feel like it would be useful to have e-cash balances per federation, makes it slightly harder to quickly sum up, but e-cash in different federations could be valued quite differently.

Reply:
I already get that in the gateway, fine having it in both places though